src/components/AddLessonAdditionalBoxes/AddLessonAdditionalBoxes.scss

```diff
@@ -0,0 +1,58 @@
+@use "../../styles/variables" as *;
+
+.add-lesson {
+    display: flex;
+    flex-direction: column;
+
+    &__input-one{
+        margin-bottom: 1.75rem !important;
+    }
+    
+    &__input-two {
+        height: 9.37rem !important;
+        width: 32rem !important;
+        margin-bottom: 1rem !important; 
+    }
+
+    &__add{
+        text-decoration: underline dotted;
+        text-underline-offset: 0.25rem;
+        color: $color-blue-navy-dark;
+        font-family: $font-secondary;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 0.8rem;
+        cursor: pointer;
+    }
+}
+
+.input[type=text], select {
+    width: 32rem;
+    height: 3rem;
+    padding: 0.75rem 1rem;
+    margin: 0.5rem 0;
+    display: inline-block;
+    box-sizing: border-box;
+    border: 1px solid #D0D5DD;
+    box-shadow: 0px 1px 2px rgba(16, 24, 40, 0.05);
+    border-radius: 0.5rem;
+    outline-color: $color-blue-navy; 
+}
```

Comment:
Please add            &:focus {              to the .input so there won't be a blue border when clicked
                                  outline: none;
                                }